docscan/client.go

```diff
@@ -243,12 +244,14 @@ func (c *Client) DeleteMediaContent(sessionID, mediaID string) error {
 }
 
 // GetSupportedDocuments gets a slice of supported documents
-func (c *Client) GetSupportedDocuments() (*supported.DocumentsResponse, error) {
+func (c *Client) GetSupportedDocuments(includeNonLatin bool) (*supported.DocumentsResponse, error) {
```

Comment:
might be a breaking change

Reply:
Preserved GetSupportedDocuments to maintain backward compatibility added different method for GetSupportedDocumentsWithNonLatin